.gitignore

```diff
@@ -48,3 +48,4 @@ coverage.xml
 
 # Environment variables
 .env
+homeworks/zlinszkygyorgy/1_introduction/homework01.py
```

Comment:
a .gitignore-ba olyan file-t ne rakja bele kérlek, amit lehetséges kéne legyen megváltoztatni. Írok egy visszajelzést részletesen is